controllers/content_items_controller.go

```diff
@@ -0,0 +1,38 @@
+package controllers
+
+import (
+	"net/http"
+	"strings"
+
+	"github.com/alphagov/publishing-api/contentstore"
+	"github.com/alphagov/publishing-api/urlarbiter"
+)
+
+type ContentItemsController struct {
+	arbiter           *urlarbiter.URLArbiter
+	liveContentStore  *contentstore.ContentStoreClient
+	draftContentStore *contentstore.ContentStoreClient
+}
+
+func NewContentItemsController(arbiterURL, liveContentStoreURL, draftContentStoreURL string) *ContentItemsController {
+	return &ContentItemsController{
+		arbiter:           urlarbiter.NewURLArbiter(arbiterURL),
+		liveContentStore:  contentstore.NewClient(liveContentStoreURL),
+		draftContentStore: contentstore.NewClient(draftContentStoreURL),
+	}
+}
+
+func (c *ContentItemsController) PutDraftContentItem(w http.ResponseWriter, r *http.Request) {
+	registerWithURLArbiterAndForward(c.arbiter, w, r, func(basePath string, requestBody []byte) {
+		doContentStoreRequest(c.draftContentStore, "PUT", strings.Replace(basePath, "draft-", "", -1), requestBody, w)
```

Comment:
I'm not sure the `strings.Replace` call is 100% correct here.  It will remove all occurrences of "draft-" from the base path.  Using a limit of 1 would improve it, but it's still a little opaque.  It might be clearer if it did a replacement from "/draft-content/" to "/content/".